.github/workflows/edge.yml

```diff
@@ -361,14 +301,15 @@ jobs:
           name: test-results-${{ steps.k8s.outputs.cluster }}
           path: ${{ github.workspace }}/tests/tests-${{ steps.k8s.outputs.cluster }}.html
         if: always()
-
-  build:
-    name: Build Docker Images
+  
+  nightly-smoke-tests:
```

Comment:
Is nightly and edge are both in single yaml now?

Reply:
Yes, but that is proving controversial so it might not stay that way (see Luca's comment above) 😄  I wanted to promote code re-use, and I dislike the overhead of having too many different workflow files that do the same thing - when we want to update the build process somehow, we have to do duplicate the effort and it increases the possibility of errors IMO. Which stages are run depend on the github event i.e. schedule/ push/ pull_request. I don't mind if I'm outvoted though if it seems harder to follow 😃

Reply:
I've stumbled across [this](https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action) the other day. Maybe we can just create a simple action for the build part so we can re-use it. I haven't looked too much into it.

Reply:
Maybe we should just remove the plus builds for now (so they stop failing 😄) and merge this?

Reply:
I returned edge and nightly to their separate ways 😄